buzzmobile/sense/inputer/inputer.py

```diff
@@ -10,7 +10,7 @@ def input_node():
     # Publish a nonsense initial value to make sure subscribers don't explode
     pub.publish("")
     while not rospy.is_shutdown():
-        new_dst = raw_input("Dest >> ")
+        new_dst = input("Dest >> ")
```

Comment:
>And running the node runs the python3 version.

Are you sure about this though? We were running this node when testing things and it wasn't failing on `raw_input` not existing. So it lead me to think that when we do `rosnode buzzmobile inputer.py`, ros runs the node using py2. That's why I was asking.